best_practises/pm.md

```diff
@@ -29,20 +27,86 @@ Agile development holds iterative design at its core. The idea is to prepare a b
+
+If you combine it all together, you will get this user story: **"As a new volunteer, I want to view the previous projects that Software For Love has worked on, so that I can gain a better understanding of the work they do to help our community."**
 
-Tackling a project in sprints helps reduce the "intimidation" factor. It's like saying you're going to bike across Canada. Anyone would be intimidated by that, but if you say you're going to bike 20 km a day, that becomes a lot easier.
+This statement easily conveys what the end functionality we would like to see is, as well as the benefit it will have. [Please view this resource for additional information regarding user stories.](https://www.atlassian.com/agile/project-management/user-stories)
 
-## Kanban Boards
+However, this should be appended with **acceptance criteria** in order to better clarify the expectations of users. Acceptance criteria should also be written from the perspective of a stakeholder, and should be as specific as possible. Use the following template when writing acceptance criteria:
 
-Kanban boards are the best way to organize sprints. You can use tools like [Trello](www.trello.com), [Github](https://docs.github.com/en/github/managing-your-work-on-github/managing-project-boards/about-project-boards).
+"Given \[context\], when \[action\], then \[result\]."
 
-Github is recommended for software projects with implementation. You can connect issues and pull requests to tasks and organization is a breeze.
+An example for the previously mentioned user story could be: "**Given** that I do not have any prior knowledge regarding Software For Love's previous endeavors, **when** I am visiting the projects page **then** I shall be presented with descriptions of all of the previous projects SFL has worked on."
 
-Trello is a great alternative if there is no Git repository associated with the project. This is great for design-only projects. Here is an example [board](https://trello.com/b/ZqN99gGN/agile-sprint-board) that you can follow.
+More informaiton on writing acceptance criteria can be found [here.](https://zepel.io/agile/acceptance-criteria-for-user-stories/)
+
+Additionally, it is also important to indicate the importance or priority of an issue when adding it to the product backlog. This will help your team be able to determine which issues they should pick up in sprint planning sessions (ie. More important stories should be completed as soon as possible, and less important stories can possibly be pushed to a later sprint.)
```

Comment:
change to 'Additionally, it is also important to indicate the priority of an...'

its weird wording to say 'important to indicate the importance'